src/type-factory/field.type-factory.ts

```diff
@@ -224,7 +232,7 @@ export function fieldTypeFactory(target: Function, metadata?: FieldMetadata, isI
     type: fieldType,
     description: metadata.description,
     args: args,
-    resolve: resolveFn,
+    ...(resolveFn ? { resolve: resolveFn } : {}),
```

Comment:
Hum... could you explain why using this approach is better?

Reply:
Because with recent changes in `graphql` itself the `resolve` field being set results an error.

Reply:
spread operator with object is just a hack, not a solution for type changes

Reply:
Agree, yet there's still an issue with `resolve`